.rubocop.yml

```diff
@@ -1,6 +1,79 @@
-inherit_from:
-  - .rubocop_rubyzip.yml
-AllCops:
-  TargetRubyVersion: 1.9
```

Comment:
What would you think to keeping the target version and setting it to `2.4` to match the minimum supported version in the Gemfile? The [docs](https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md#setting-the-target-ruby-version) say that it will otherwise take its value from `.ruby-version`, which might be different. (This resulted in a warning in the gemspec for me, since I had `.ruby-version` set to a later version locally.)

Reply:
Good call; will add. I've taken to setting `.ruby-version` to whatever is in the gemspec so completely missed this. Interestingly `rubocop --auto-gen-config` seems to omit that if the two match up.